spaceship/lib/spaceship/connect_api/models/app.rb

```
@@ -0,0 +1,508 @@
+
+      ESSENTIAL_INCLUDES = [
+        "appStoreVersions"
+      ].join(",")
+
+      def self.type
+        return "apps"
+      end
+
+      def removed_from_sale?
+        ready_for_distribution = get_ready_for_distribution_app_store_version(includes: nil)
+        return true if ready_for_distribution.nil?
```

RuboCop (1.88.2)

[convention] Add empty line after guard clause.

(Layout/EmptyLineAfterGuardClause)